ts/screens/onboarding/TosScreen.tsx

```diff
@@ -193,7 +189,7 @@ class TosScreen extends React.PureComponent<Props, State> {
             : I18n.t("onboarding.tos.headerTitle")
         }
       >
-        <SafeAreaView style={styles.webViewContainer}>
+        <View style={styles.webViewContainer}>
```

Comment:
I would not replace `SafeAreaView` with a simple `View` cause we need the SafeAreaView to add some space during the onboarding flow on the footer buttons, space needed on iPhone X like smartphones.

Maybe just replacing the `SafeAreaView` component from react-navigation with the one from react-native may fix both requirements?

here my attempt: cc212d9

<img src="https://user-images.githubusercontent.com/3959405/94578265-d1d9ae80-0277-11eb-8ad9-0afb3d9f412a.png" height="550"/>

Reply:
Followed your advice and it works in both situations